### PR TITLE
expose thing definition

### LINKED ISSUE #127: [JS]: Thing is missing the definition field
It is returned in the response, but it is never mapped.

### PR DESCRIPTION
**Pull request by vhdirk**
Adds a `_definition` field to the thing model.

Fixes #127

### CODE REVIEW

**Comment by ffendt** [comment]
Hi @vhdirk,
thanks for this contribution. Would you please add the definition to the model test?

Have you thought about also creating the respective handles for GET/PUT/DELETE of a definition (see e.g. [GET definition](https://www.eclipse.org/ditto/http-api-doc.html#/Things/get_things__thingId__definition)?

**Comment by vhdirk** [comment]
@ffendt No, I hadn't. But I did now :)

**Comment by vhdirk** [comment]
@ffendt I'm not sure why the tests are failing still?

File: javascript/lib/api/tests/client/http/things.http.spec.ts [hunk 1]
```diff
@@ -217,6 +228,18 @@ describe('Http Things Handle', () => {
     });
   });
 
+  it('updates the definition', () => {
```
Comment (ffendt) [inline javascript/lib/api/tests/client/http/things.http.spec.ts:231]: Please also add the corresponding 'creates the definition'-test

**Review by ffendt — CHANGES_REQUESTED** [review]
Looks good so far, except the remaining linter errors

File: javascript/lib/api/tests/client/http/things.http.spec.ts (hunk 1 above)
Comment (vhdirk) [reply javascript/lib/api/tests/client/http/things.http.spec.ts]: I can't yet. That would need #66 to be merged.

Comment (vhdirk) [reply javascript/lib/api/tests/client/http/things.http.spec.ts]: Done

**Review by ffendt — APPROVED** [review]

**Comment by ffendt** [comment]
Thanks @vhdirk, looks good to me. I'll also add this to the set of features for 3.0.0 👍

**Pull request merged**